website/project/model.py

```diff
@@ -321,6 +350,13 @@ def create(cls, auth, **kwargs):
             save=False,
         )
 
+        new_mentions = get_valid_mentioned_users_guids(comment, comment.node.contributors)
+
+        if new_mentions:
+            project_signals.mention_added.send(comment, new_mentions=new_mentions, auth=auth)
+            comment.ever_mentioned.extend(new_mentions)
+            comment.save()
+
```

Comment:
Can we rework this to only save once? We should be able to make it look a lot like the `edit` case, no?